go.mod

```diff
@@ -3,105 +3,98 @@ module github.com/minio/mc
 go 1.21
```

Comment:
Changes here in go.mod are cleanup, sorting imports into 2 sections `indirect` and direct.

Reply:
you don't need to manually do it @marktheunissen you should let go do it for you via `go mod tidy -compat=1.21`

Reply:
`go mod tidy -compat=1.21` doesn't have any effect on go.mod on `master` branch. This is probably because it does not rejoin the require sections https://github.com/golang/go/issues/56471

Once it's manually rejoined, then `go mod tidy` should work ok.

Look it's mostly cosmetic fix, but at the moment the go.mod has dependencies all out-of-order and the tidy doesn't work, see screenshot:

<img width="684" alt="Screenshot 2024-07-03 at 14 53 29" src="https://github.com/minio/mc/assets/453297/fcc13f06-2bf8-430b-855a-e7e31882b2f3">

The charmbracelet deps should rather be together, indirect should be grouped together. 

Unless I'm missing something?